uncaught exception: error(type_error(callable,(false,1)),main/0)
